src/sentry/templates/sentry/bases/react.html

```diff
@@ -9,7 +9,7 @@
       <div class="loading-mask"></div>
 
       <div class="loading-indicator">
-        <video autoplay disablePictureInPicture loop height="150">
+        <video muted autoplay disablePictureInPicture loop playsinline height="150" id="animated-loader">
```

Comment:
do you really need `id="animated-loader"`?

Reply:
No, I'll take that out.